### ISSUE TITLE
AdViewController  mRefreshTimeMillis incorrectly intialized

### ISSUE THREAD

**Issue by pablo-navarro** [mopub/mopub-android-sdk#148]
In AdViewController class  mRefreshTimeMillis should not be initialized or if it's initialized its value should be reassigned in onAdLoadSuccess, Example:

On the DashBoard a user sets the update Interval to 0, but on AdViewController a default value is assigned to mRefreshTimeMillis:

```java

public AdViewController(Context context, MoPubView view) {

....
mRefreshTimeMillis = DEFAULT_REFRESH_TIME_MILLISECONDS;
....

}
```

Then onAdLoadSuccess this behavior is not corrected and the even if the user has no declared a refresh time interval, the ad is refreshed with the default interval.

```java
void onAdLoadSuccess(@NonNull final AdResponse adResponse) {
....
/* This is wrong, mRefreshTimeMillis should be assigned to refreshTime even if refreshTime value is null or the refresh will be triggered*/

if (refreshTime != null) {
        mRefreshTimeMillis = refreshTime;
}
....
}
```

**Comment by MoJRen** [comment]
thanks for the report @pablo-navarro this will be fixed in an upcoming release

**Comment by joeblubaugh** [comment]
This was fixed in release 3.5.0
